api/go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/ceph/ceph-csi-operator/api
+
+go 1.22.5
```

Comment:
Why the bump from 1.22.0 to 1.22.5?
And if we do want to do it, lets have a separate PR for it

Reply:
fixed, moved it back. and set it up just like the parent package